README.md

```diff
@@ -4,3 +4,12 @@ This repo contains our pod's 2020 STEP Capstone project.
 
 Authors: handeland@google.com, jevingu@google.com, mnuzen@google.com 
 Reviewers: arunkaly@, promanov@
+
+
+## To use geolocation: 
+
+1. Download `GeoLite2-City.mmdb` from GDrive
+2. Put in repo under `data/GeoLite2-City.mmdb`
```

Comment:
Try to use the .git folder for this or some other folder that you can mark in `.gitignore` so you don't accidentally commit this file.

Reply:
Yep in gitignore